[2023-07-26 23:55:00,135] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_ manual__2023-07-26T23:54:55.297911+00:00 [queued]>
[2023-07-26 23:55:00,157] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_con_conexion_postgres.crear_tabla_ manual__2023-07-26T23:54:55.297911+00:00 [queued]>
[2023-07-26 23:55:00,159] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-26 23:55:00,161] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-26 23:55:00,166] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-26 23:55:00,225] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): crear_tabla_> on 2023-07-26 23:54:55.297911+00:00
[2023-07-26 23:55:00,234] {standard_task_runner.py:52} INFO - Started process 4930 to run task
[2023-07-26 23:55:00,247] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_con_conexion_postgres', 'crear_tabla_', 'manual__2023-07-26T23:54:55.297911+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_paises.py', '--cfg-path', '/tmp/tmps285mlky', '--error-file', '/tmp/tmp9qwvy_gt']
[2023-07-26 23:55:00,250] {standard_task_runner.py:80} INFO - Job 134: Subtask crear_tabla_
[2023-07-26 23:55:00,446] {task_command.py:371} INFO - Running <TaskInstance: dag_con_conexion_postgres.crear_tabla_ manual__2023-07-26T23:54:55.297911+00:00 [running]> on host 484a5a9deba0
[2023-07-26 23:55:00,972] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Belenpintor
AIRFLOW_CTX_DAG_ID=dag_con_conexion_postgres
AIRFLOW_CTX_TASK_ID=crear_tabla_
AIRFLOW_CTX_EXECUTION_DATE=2023-07-26T23:54:55.297911+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-07-26T23:54:55.297911+00:00
[2023-07-26 23:55:01,014] {base.py:68} INFO - Using connection ID 'redshift_belen' for task execution.
[2023-07-26 23:55:02,295] {dbapi.py:231} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS public.ciudades (
    city VARCHAR PRIMARY KEY,
    "Housing" DECIMAL(10, 2),
    "Cost of Living" DECIMAL(10, 2),
    "Startups" DECIMAL(10, 2),
    "Venture Capital" DECIMAL(10, 2),
    "Travel Connectivity" DECIMAL(10, 2),
    "Commute" DECIMAL(10, 2),
    "Business Freedom" DECIMAL(10, 2),
    "Safety" DECIMAL(10, 2),
    "Healthcare" DECIMAL(10, 2),
    "Education" DECIMAL(10, 2),
    "Environmental Quality" DECIMAL(10, 2),
    "Economy" DECIMAL(10, 2),
    "Taxation" DECIMAL(10, 2),
    "Internet Access" DECIMAL(10, 2),
    "Leisure & Culture" DECIMAL(10, 2),
    "Tolerance" DECIMAL(10, 2),
    "Outdoors" DECIMAL(10, 2),
    "process_date" TIMESTAMP DISTKEY
);, parameters: None
[2023-07-26 23:55:02,487] {postgres.py:94} INFO - INFO:  Relation "ciudades" already exists and will be skipped

[2023-07-26 23:55:02,549] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_con_conexion_postgres, task_id=crear_tabla_, execution_date=20230726T235455, start_date=20230726T235500, end_date=20230726T235502
[2023-07-26 23:55:02,740] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-26 23:55:03,000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
